conn.go

```diff
@@ -658,17 +671,26 @@ func (c *Conn) ReadMessage(maxBytes int) (Message, error) {
 // gives the minimum and maximum number of bytes that it wants to receive from
 // the kafka server.
 func (c *Conn) ReadBatch(minBytes, maxBytes int) *Batch {
+	cfg := BatchReadConfig{
```

Comment:
This temporary variable doesn't seem useful, how about just inlining the declaration in argument to the method call?